### ISSUE TITLE
npcap fails to download

### ISSUE THREAD

**Issue by Destroyer117a** [nmap/npcap#180]
i have windows 10 and i am trying to download  npcap through npcap 0.9991 installer for windows.
when i try to install it, it gives me this message

![npcap](https://user-images.githubusercontent.com/65191897/83533474-ad0b3f80-a500-11ea-8ca7-1cf4b5b7eee2.PNG)

**Comment by dmiller-nmap** [comment]
Thanks for the report. Please try Npcap 0.9992, released today. If you still have problems, please include the `NPFInstall.log` file from the Npcap installation folder. Otherwise, please let us know so we can close this issue.

**Comment by Destroyer117a** [comment]
Thanks for Your Support. I will send the NPFInstall.log file and also try out the new version.

By the way, the new version can be found on the website, right?

**Comment by dmiller-nmap** [comment]
Yes, you can find the latest version at https://npcap.org/#download . We just released Npcap 0.9993 to address several bugs, so be sure to use that version instead.
